deploy/distrib/src/main/resources/tomcat/conf/logging.properties

```diff
@@ -23,7 +23,7 @@ handlers = java.util.logging.ConsoleHandler, 1catalina.org.apache.juli.FileHandl
 .handlers = 1catalina.org.apache.juli.FileHandler, java.util.logging.ConsoleHandler
 
 # Allow logging to trace the actual logger producer class:
-java.util.logging.SimpleFormatter.format=%1$tF %1$tT %3$s %n%4$s: %5$s%6$s%n
+java.util.logging.SimpleFormatter.format=%1$tF %1$tT.%1$tL %1$tz %3$s %2$s %n%4$s: %5$s%6$s%n
```

Comment:
Example is in the PF

Log file example:
2015-04-25 23:57:15.850 +0200 org.bonitasoft.engine.scheduler.impl.BonitaJobStoreCMT org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler manage 
FINE: MisfireHandler: scanning for misfires...